app/helpers/document_helper.rb

```diff
@@ -1,16 +1,9 @@
 module DocumentHelper
   include ApplicationHelper
-  include CountryHelper
-  include DocumentCollectionHelper
-  include MinisterialRolesHelper
-  include PolicyGroupsHelper
-  include RoleAppointmentsHelper
-  include TopicsHelper
-  include TranslationHelper
 
-  def edition_page_title(edition)
-    edition.withdrawn? ? "[Withdrawn] #{edition.title}" : edition.title
-  end
+  MS_WORD_DOCUMENT_HUMANIZED_CONTENT_TYPE = "MS Word Document".freeze
```

Comment:
Could these be moved into the locale files?

Reply:
Ooh yes they should, good spot. Same could be said about the [other humanised content types in the helper](https://github.com/alphagov/whitehall/blob/984764296ac058ac673e58652be79ef23fb6b184/app/helpers/document_helper.rb#L23-L62). 

Will do this in a separate PR!